cw/tvla.py

```diff
@@ -48,6 +50,38 @@ def bit_count(int_no):
     return c
 
 
+def plot_fixed_vs_random(traces, leakage):
```

Comment:
I suggest to change this name into something more precise, e.g. `plot_stats_fvsr` or `plot_statistics_fvsr`.

Reply:
Sounds good. Let me do this.

Reply:
Done.